packages/askui-nodejs/example_projects_templates/typescript_jest/test/my-first-askui-test-suite.test.ts

```diff
@@ -1,15 +1,14 @@
 import { aui } from './helper/jest.setup';
 
 describe('jest with askui', () => {
-  it('should click on text', async () => {
-    // Run this to see what askui annotates
+
+  it('should generate an interactive annotation', async () => {
     await aui.annotateInteractively();
+  });
 
-    await aui.moveMouse(0, 0).exec();
+  it('should click on my element', async () => {
```

Comment:
should we disable this and let the user enable it after replacing the comment?

Reply:
I thought about it and decided against it as not everybody will be familiar with the concept of using `xit` to ignore a test case.